org.jcryptool.core.introduction/src/org/jcryptool/core/introduction/views/AlgorithmInstruction.java

```diff
@@ -260,27 +264,30 @@ public void createPartControl(Composite parent) {
 		gl_content.horizontalSpacing = 0;
 		gl_content.verticalSpacing = 0;
 		content.setLayout(gl_content);
+		content.setLayoutData(new GridData(SWT.FILL, SWT.FILL, true, true));
```

Comment:
this is needed. the outer composite neets to fill the available space so there is room to "center within".